staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -63,6 +64,10 @@ type proxyHandler struct {
 	serviceResolver ServiceResolver
 
 	handlingInfo atomic.Value
+
+	// egressSelector is the egress selector to use to communicate to the api aggregator
```

Comment:
This comment doesn't seem very clear. Perhaps something like "egressSelector selects the proper egress dialer to communicate with the api aggregator"?

Reply:
:+1: (however right now it's directly to the cluster, no choice :thinking: )

Reply:
okay looks like the api aggregator only communicates through services. @Jefftree or @caesarxuchao can you confirm?

Reply:
Currently it is, it only supports [service](https://github.com/kubernetes/kubernetes/blob/bbc035a66cacc345dd54676e6c4e1f0389dbcb5a/staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/types.go#L52). It could support a URL in the future.

Reply:
So I'll just leave it as is for now